x-pack/auditbeat/processors/add_session_metadata/add_session_metadata.go

```diff
@@ -0,0 +1,194 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package add_session_metadata
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/elastic/elastic-agent-libs/mapstr"
+
+	"github.com/elastic/elastic-agent-libs/monitoring"
+
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/pkg/processdb"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/pkg/procfs"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/provider"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/provider/ebpf_provider"
+
+	"github.com/elastic/beats/v7/libbeat/beat"
+	"github.com/elastic/beats/v7/libbeat/processors"
+	"github.com/elastic/elastic-agent-libs/config"
+	"github.com/elastic/elastic-agent-libs/logp"
+)
+
+const (
+	processorName = "add_session_metadata"
+	logName       = "processor." + processorName
+)
+
+var reg *monitoring.Registry
```

Comment:
This is indeed not referred to anywhere else than the init assignment. How is this used?

Reply:
I've removed this